src/plugins/user-activity-events/client.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-invalid-this */
+'use strict';
+
+function throttle(fn) {
+	var last, deferTimer;
+	return function() {
+		var context = this;
+		//only fire the event if it has been at least 10 seconds since previous fire
+		var threshhold = 10000;
+		var now = +new Date();

```

Comment:
Did you mean to have the `+` here? There's also `var now = Date.now();` [reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now)